src/auth/components/loginForm/componentsForLoginForm.tsx

```diff
@@ -63,3 +55,11 @@ export const AlertForm: React.FC<IAlertForm> = ({ message }: IAlertForm) => {
     </Alert>
   );
 };
+
+export const ButtonEl: React.FC<IButtonEl> = ({ text, ...props }) => {
+  return (
+    <Button variant="contained" type="submit" {...props}>
+      {text}
+    </Button>
+  );
+};
```

Comment:
Button element does not belong only to `LoginForm`. It would be anywhere in the app, at least that's what I think looking at the name `ButtonEl`. So please move this component to some place for generic components that are not related to a particular piece of UI

Reply:
done